### PR TITLE
Per org maxcontacts service

### PR DESCRIPTION
**Pull request by schuyler1d**
This adds some per-organization features (currently, only settable with direct database access) including which service is used, and if there is a maxContacts count.

This addresses #279, because we can now make QAers admins or set the maxContacts for only our own orgs to actually send messasges on a service or send them but have a maxContacts setting.

For the same purpose, we also have a `fakeservice` that joins twilio and nexmo for DEFAULT_SERVICE as an option, which means it won't try to send through twilio, etc and you can use, e.g. just the `/admin/1/campaigns/5/send-replies` url to fake-send replies.  This will also be useful for testing sans-service.

### CODE REVIEW

**Review by sandramchung — APPROVED** [review]
👍  fc49a7d916fae99bcc60ff0bfdaed5b4e4a8d545

File: src/server/api/schema.js [hunk 1]
```diff
@@ -322,7 +323,7 @@ async function editCampaign(id, campaign, loaders, user) {
 
 const rootMutations = {
   RootMutation: {
-    sendReply: async (_, { id, message }, { loaders }) => {
+    sendReply: async (_, { id, message }, { user, loaders }) => {
       if (process.env.NODE_ENV !== 'development') {
```
Comment (sandramchung) [inline src/server/api/schema.js]: I imagine we'll want to use this to test in a production-like environment - so we'd have to deploy it in dev mode to do that?

Comment (schuyler1d) [reply src/server/api/schema.js]: ah, i missed that.  i'm going to make is so development node env OR have correct access

**Comment by mathemagica** [comment]
Belatedly, thanks for this- this'll make testing much easier!

**Pull request merged**